src/js/tabs/firmware_flasher.js

```diff
@@ -1082,7 +1082,8 @@ TABS.firmware_flasher.flashingMessage = function(message, type) {
             break;
     }
     if (message != null) {
-        if (i18next.exists(message)) {
+        // If we're passed an item that could be a localization message key, try looking it up.
+        if (!message.includes(':') && i18next.exists(message)) {
```

Comment:
I don't think that making 'best effort' determinations if something is a message or a key here is a good approach. Better approachese are:
- only accept either localised strings, or localisation keys for `message`;
- use a string for localised strings, `{ key: <localisation key> }` for localisation keys.

Reply:
The first option looks the best. This extra fluff was probably related to me trying to change languages everywhere.